src/components/organisms/LeftSection/LeftSection.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from "react";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import Divider from "@mui/material/Divider";
+import ListItemText from "@mui/material/ListItemText";
+import ListItemAvatar from "@mui/material/ListItemAvatar";
+import Avatar from "@mui/material/Avatar";
+import Typography from "@mui/material/Typography";
+import { Box } from "@mui/material";
+import TextField from "@mui/material/TextField";
+import styles from "./LeftSection.module.css";
+import Button from "@mui/material/Button";
+import { AddNewChatModal } from "../../molecules/AddNewChatModal/AddNewChatModal";
+import { useState } from "react";
+
+export const LeftSection = () => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
```

Comment:
To już nie obowiązkowo, ale super by było, gdybyście zawsze określali też typy tego co zwraca funkcja:

```suggestion
  const handleClickOpen = () : void => {
    setOpen(true);
  };

  const handleClose = () : void => {
    setOpen(false);
  };
```

Reply:
Racja